content/influxdb/cloud-serverless/query-data/influxql/execute-queries/influxdb-v1-api.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Use the InfluxDB v1 HTTP query API to query with InfluxQL
+seotitle: Use InfluxQL and InfluxDB v1 HTTP query API
+description: >
+  Use the InfluxDB v1 HTTP query API to query data in InfluxDB Cloud Serverless
+  with InfluxQL.
+weight: 401
+menu:
+  influxdb_cloud_serverless:
+    parent: influxql-execute-queries
+    name: Use the v1 query API
+influxdb/cloud-serverless/tags: [query, influxql, python]
+list_code_example: |
+  ```sh
+  curl --get https://cloud2.influxdata.com/query \
+    --header "Authorization: Token DATABASE_TOKEN" \
+    --data-urlencode "db=DATABASE_NAME" \
+    --data-urlencode "q=SELECT * FROM home"
+  ```
+---
+
+Use the InfluxDB v1 HTTP query API to query data in {{< cloud-name >}}
+with InfluxQL.
+
+The examples below use **cURL** to send HTTP requests to the InfluxDB v1 HTTP API,
+but you can use any HTTP client.
+
+{{% warn %}}
+#### InfluxQL feature support
+
+InfluxQL is being rearchitected to work with the InfluxDB IOx storage engine.
+This process is ongoing and some InfluxQL features are still being implemented.
+For information about the current implementation status of InfluxQL features,
+see [InfluxQL feature support](/influxdb/cloud-serverless/reference/influxql/feature-support/).
+{{% /warn %}}
+
+Use the v1 `/query` endpoint and the `GET` request method to query data with InfluxQL:
+
+{{< api-endpoint endpoint="https://cloud2.influxdata.com/query" method="get" api-ref="/influxdb/cloud-serverless/api/#tag/Query" >}}
+
+Provide the following with your request:
+
+- **Headers:**
+  - **Authorization:** `Bearer DATABASE_TOKEN`
+- **Query parameters:**
+  - **db**: Database to query
```

Comment:
> This one is tricky, because in InfluxQL, this actually the database. Requests are mapped through DBRPs.

Yeah, I assumed that was it, but in Cloud2 and Serverless we still refer to the objects as "Buckets".